src/control/server/instance.go

```diff
@@ -439,3 +439,9 @@ func (srv *IOServerInstance) CallDrpc(module, method int32, body proto.Message)
 
 	return makeDrpcCall(dc, module, method, body)
 }
+
+func (srv *IOServerInstance) BioErrorNotify(bio *srvpb.BioErrorReq) {
+
+	srv.log.Debugf("I/O server instance %d (target %d) has detected blob I/O error! %v",
```

Comment:
Is DEBUG the right log level for this? I would think it should be ERROR or at least INFO... People running in production shouldn't be running with debug logging enabled, generally speaking. How often do these kinds of error happen, and what should the admin do about them?

If there is a known resolution, I'd suggest adding a new fault for this situation, but we can do that in a follow-up. My main concern in this PR is that these messages will never be seen by anyone unless debug logging is enabled.

Reply:
good point, will change to ERROR